doc/faq.rst

```diff
@@ -165,6 +165,66 @@ Using Spyder
+       |C:\Users\<username>\Miniconda    | /Users/<username>/opt/miniconda |
+       +---------------------------------+---------------------------------+
+       |C:\Users\<username>\Anaconda3    | /Users/<username>/opt/anaconda3 |
+       +---------------------------------+---------------------------------+
+       |C:\Users\<username>\Miniconda3   | /Users/<username>/opt/miniconda3|
+       +---------------------------------+---------------------------------+
+       |C:\Anaconda                      |   /opt/anaconda                 |
+       +---------------------------------+---------------------------------+
+       |C:\Miniconda                     |   /opt/miniconda                |
+       +---------------------------------+---------------------------------+
+       |C:\Anaconda3                     |   /opt/anaconda3                |
+       +---------------------------------+---------------------------------+
+       |C:\Miniconda3                    |   /opt/miniconda3               |
+       +---------------------------------+---------------------------------+
+       |C:\ProgramData\Anaconda          |                                 |
+       +---------------------------------+---------------------------------+
+       |C:\ProgramData\Miniconda         |                                 |
+       +---------------------------------+---------------------------------+
+       |C:\ProgramData\Anaconda3         |                                 |
+       +---------------------------------+---------------------------------+
+       |C:\ProgramData\Miniconda3        |                                 |
+       +---------------------------------+---------------------------------+
+
+   .. _Miniconda: https://docs.conda.io/en/latest/miniconda.html
+
+   Then, you need to create a new conda environment with the modules that you want to use with spyder and include ``spyder-kernels`` in it. For example, if you want to use ``scikit-klearn``, open your terminal or Anaconda prompt on Windows and run the following commands:
+
+   .. code-block:: bash
+
+      conda create -n spyder-env -y
+      conda activate spyder-env
+      conda install spyder-kernels scikit-learn -y
+
+   Finally, you need to connect Spyder with this new environment by changing Spyder’s default Python interpreter. To do this, click the name of the current environment in the status bar, and then click :guilabel:`Change default environment in Preferences`.
+
+   .. image:: /images/faq/faq-change-environment.png
+      :alt: Change default environment in Preferences option in status bar
+
+   This will open the :guilabel:`Preferences` dialog in the :guilabel:`Python interpreter` section. Here, select the option :guilabel:`Use the following Python interpreter`, and use the dropdown below to select your preferred environment. If it is not listed, use the text box or the :guilabel:`Select file` button to enter the path to the Python interpreter you want to use.
```

Comment:
```suggestion
   This will open our :guilabel:`Preferences` in the :guilabel:`Python interpreter` section. Here, select the option :guilabel:`Use the following Python interpreter`, and use the dropdown below to select your preferred environment. If it is not listed there, use the text box or the :guilabel:`Select file` button to enter the path to the Python interpreter you want to use.
```

- No need to mention that Preferences is a dialog. I think that's a technical detail that users are not interested in.
- I think `there` is missing.